lib/client/auth-code.js

```diff
@@ -18,7 +19,13 @@ module.exports = function (config) {
     params.response_type = 'code';
     params.client_id = config.clientID;
 
-    return config.site + config.authorizationPath + '?' + qs.stringify(params);
+    var isAbsolute = String(Url.parse(config.authorizationPath).protocol)
```

Comment:
Right now the test suit is only covering high-level cases. I´ll try to improve that in the future. Thanks for this.